pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -794,6 +795,11 @@ func (nc *Controller) doEvictionPass(ctx context.Context) {
 // if not, post "NodeReady==ConditionUnknown".
 // This function will taint nodes who are not ready or not reachable for a long period of time.
 func (nc *Controller) monitorNodeHealth(ctx context.Context) error {
+	start := time.Now()
+	defer func() {
```

Comment:
Why do we need to defer the metrics?

Reply:
It's not strictly required, but it avoids needing to explicitly handle the different early return cases